covid19-etl/etl/idph_facility.py

```diff
@@ -117,14 +119,15 @@ def parse_facility(self, date, facility):
         facility_name = facility["FacilityName"]
         confirmed_cases = facility["confirmed_cases"]
         deaths = facility["deaths"]
-        status = facility["status"]
+        status = facility.get("status", "Open")
```

Comment:
why is the default "Open"? shouldn't we use None if we don't have the data?

Reply:
This is `None` now, I have checked with Sean.